R/estimate_secondary.R

```diff
@@ -238,15 +238,17 @@ secondary_opts <- function(type = "incidence", ...) {
       historic = 1,
       primary_hist_additive = 1,
       current = 0,
-      primary_current_additive = 0
+      primary_current_additive = 0,
+      estimate_baseline = 0
```

Comment:
user control to turn this on or off independently? Could think about doing so by passing the mean intercept and having 0 be the setting that turns it off (so endemic = c(100, 10) is a prior of N(100, 10), and endemic = 0 turns it off)